docs/phenotype.rst

```diff
@@ -72,7 +72,8 @@ For many medical use cases, either the Age sub-element or an :ref:`ontology clas
 
 evidence
 ~~~~~~~~
-This element can contain one or more :ref:`Evidence <rstevidence>` elements that specify how the phenotype was determined.
+This element is required and must contain one or more :ref:`Evidence <rstevidence>` elements
```

Comment:
Should the evidence field be `required`? I'd rather `recommended`.

Reply:
I have altered to make evidence recommended